lib/workers/global/config/parse/file.ts

```diff
@@ -78,6 +79,22 @@ export async function getConfig(env: NodeJS.ProcessEnv): Promise<AllConfig> {
     logger.debug('No config file found on disk - skipping');
   }
 
+  if (is.nonEmptyObject(config.processEnv)) {
+    // should begin and end with capital alphabets + use underscore as separator
+    const keyRegex = regEx(/^[A-Z]+(_[A-Z]+)*$/);
```

Comment:
Where is this from? Leave a reference to the spec

Reply:
I don't think there's a spec/ref to this. I took it from the env names we used for our config options.
like `labels` -> `RENOVATE_LABELS` : separated by `_`and all capitals

Reply:
Just realized that if users use their own prefix instead of `RENOVATE_`, then they could include lowercase letter as well.

Don't think checking the key name makes sense now.

Reply:
This feature is not intended for renovate's own env. It's intended for other env so should allow all valid env variable name formats